frontend/src/App.tsx

```diff
@@ -12,6 +13,7 @@ function App() {
                 <Routes>
                     <Route path="/" element={<Login />} />
                     <Route path="*" element={<NotFound />} />
+                    <Route path="roles" element={<Roles />} />
```

Comment:
did you pull the main into your branch after I merged my homepage?

Reply:
done